src/components/molecules/UserProfilePicture/UserProfilePicture.tsx

```diff
@@ -152,12 +160,15 @@ const UserProfilePicture: React.FC<UserProfilePictureProp> = ({
     reactions,
     muteReactions,
     isAudioEffectDisabled,
+    showNametags,
+    currentVenue,
   ]);
 };
 
 UserProfilePicture.defaultProps = {
   avatarClassName: "profile-icon",
   miniAvatars: false,
+  showNametags: true,
 };
```

Comment:
Ah.. I see that we're using another non-standard (for our codebase) pattern for defaults here. I believe our preference is to add them using the 'inline' ES6 pattern that I suggested above.

Reply:
Just to ensure I'm following correctly, after implementing the suggestion you made above, I should simply delete this line?

Reply:
Yup. Though if you want to take it one step further, you can move all of these defaults to the same place I suggested the other change, and then remove `UserProfilePicture.defaultProps` entirely.